### PR TITLE
[FlexibleHeader] Added section for iOS 13 collection considerations

### PR DESCRIPTION
**Pull request by randallli**
Added docs for iOS 13 order of operations nuance.

### CODE REVIEW

**Comment by randallli** [comment]
i couldn't run 
`./scripts/generate_readmes`
because my jazzy install isn't working.
I will try again on Monday.

**Comment by romoore** [comment]
I was able to install it using gem

```bash
sudo gem install jazzy
```

What steps have you tried so far?

**Comment by randallli** [comment]
> I was able to install it using gem
> 
> ```shell
> sudo gem install jazzy
> ```
> 
> What steps have you tried so far?

`sudo gem install jazzy`
produces some errors for me:

Building native extensions.  This could take a while...
ERROR:  Error installing jazzy:
	ERROR: Failed to build gem native extension.

    current directory: /Library/Ruby/Gems/2.3.0/gems/redcarpet-3.5.0/ext/redcarpet
/System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/bin/ruby -r ./siteconf20190930-1802-62dlhb.rb extconf.rb
mkmf.rb can't find header files for ruby at /System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/include/ruby.h

extconf failed, exit code 1

Gem files will remain installed in /Library/Ruby/Gems/2.3.0/gems/redcarpet-3.5.0 for inspection.

File: components/FlexibleHeader/README.md [hunk 1]
```diff
@@ -261,6 +261,14 @@ self.headerViewController.headerView.trackingScrollView = scrollView;
 
 `scrollView` might be a table view, collection view, or a plain UIScrollView.
 
+#### iOS 13 Collection Considerations
+
+iOS 13 changed the behavior of the `contentInset` of a collection view by triggering a layout.
+This may affect your app if you have not yet registered cells for reuse yet. Our recomendation is
+to use view controller composition by making your collection view a child view controller. If this
```
Comment (romoore) [inline components/FlexibleHeader/README.md]: Nit: s/by making your collection view/by making your collection view controller/

File: components/FlexibleHeader/README.md [hunk 2]
```diff
@@ -261,6 +261,14 @@ self.headerViewController.headerView.trackingScrollView = scrollView;
...
+is not possible then ensuring your order of opperations has cells registered before setting the
```
Comment (romoore) [inline components/FlexibleHeader/README.md]: Nit: Suggested rewrite, "...then ensure the correct order of operations by registering cell reuse identifiers before setting the Flexible Header's `trackingScrollView`."

**Review by romoore — APPROVED** [review]
LGTM with nits/copyedits.

File: components/FlexibleHeader/README.md (hunk 1 above)
Comment (randallli) [reply components/FlexibleHeader/README.md]: Done

File: components/FlexibleHeader/README.md (hunk 2 above)
Comment (randallli) [reply components/FlexibleHeader/README.md]: Done.

**Pull request merged**